BUILD

```diff
@@ -19,3 +19,5 @@ lines_sorted_test(
     error_message = "Authors must be sorted by first name",
     file = "AUTHORS",
 )
+
+load("@io_bazel_rules_go//go:def.bzl", "go_prefix")
```

Comment:
this appears to be a bug that it adds this not detecting it as the same as the first line in the file.